src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -59,7 +59,7 @@ public void start(Stage primaryStage) {
             mainWindow = new MainWindow(primaryStage, config, prefs, logic, security);
             mainWindow.show(); //This should be called before creating other UI parts
             //Gets rid of the login window when testing
-            if (security.getAuthentication()) {
+            if (true) { // TODO : security.getAuthentication()
                 mainWindow.fillInnerParts();
```

Comment:
There has to be a better way to handle skipping the login screen... Hopefully we remember to remove this before v1.3